types/api/public-offer-v2.d.ts

```diff
@@ -551,6 +506,70 @@ export namespace PublicOfferV2 {
     type: LeTourOfferType;
   }
 
+  interface LeOfferSkeleton {
+    id: string;
+    name: string;
+    slug?: string;
+  }
+
+  interface TourV2Offer extends LeOfferSkeleton {
+    defaultOptionId: string; // Fk which tells the FE which tourOption to display on the offer page
+    tourOptions: Record<string, TourOption>; // Contains the details of all the default option
+    // and all other options for this offer. Required for the "Other packages for this tour" component. Refer to latest designs.
+    type: "tour_v2";
+    source: Tour.Source;
+    brand: Tour.Brand;
+    monthlyPrices: Array<{
+      // Array of monthly prices for the default tourOption
+      year: string;
+      month: string;
+      price: number;
+    }>;
+  }
```

Comment:
Once you have `options` on your base, you now override them with your own:
```suggestion
interface TourDescription {
   id: string;
   copy: SeasonCopy;
   season: Record<string, Season>;
}

interface TourOption extends OptionBase {
   tourDescriptionId: string;
   departureId: string;
}


  interface TourV2Offer extends LeOfferSkeleton {
    defaultOptionId: string; // Fk which tells the FE which tourOption to display on the offer page
    tourDescriptions: Record<string, tourDescriptions>; // Contains the details of all the default option
    // and all other options for this offer. Required for the "Other packages for this tour" component. Refer to latest designs.
    type: "tour_v2";
    source: Tour.Source;
    brand: Tour.Brand;
    monthlyPrices: Array<{
      // Array of monthly prices for the default tourOption
      year: string;
      month: string;
      price: number;
      options: Array<TourOption>;
    }>;
  }
```